spec/classes/grafana_spec.rb

```diff
@@ -162,7 +163,8 @@
       context 'with parameter install_method is set to archive' do
         let(:params) do
           {
-            install_method: 'archive'
+            install_method: 'archive',
```

Comment:
so if somebody uses the archive install method, he needs to set the parameter 'version' explicitly? I'm not sure if this is a good idea.

Reply:
I get what you mean, but managing the version within the module is also not a good idea. My assumption was: Most users are going to use packages instead of the archive method, so this changes makes it easier for the majority of the users while still allowing the use the archive method.

What do you suggest?

Reply:
I think the best approach here is to set the `version` parameter depending on the `install_method` parameter, that should solve the issue. Also I prefer to keep the default to `archive`, a change would require a major release.